package.json

```diff
@@ -116,9 +119,10 @@
     "loader.js": "^4.7.0",
     "npm-run-all": "^4.1.5",
     "prettier": "^2.5.1",
-    "qunit": "^2.17.2",
-    "qunit-dom": "^1.6.0",
-    "typescript": "^4.4.4"
+    "qunit": "^2.20.0",
```

Comment:
`qunit` is in the `peerDependencies`, do we keep here too?

Reply:
Yep! yarn does not auto-install peers (pnpm does 🙈 ) so have to list it in both keys